### PR TITLE
Feature/ELG-15/APIs for refrigerated storage cabinets for professional use

### PR DESCRIPTION
**Pull request by FernandoRamirez-De-Aguilar**
APIs for refrigerated storage cabinets for professional use

Basic implementation of reading allowed API values from enums

### CODE REVIEW

File: src/main/java/uk/gov/beis/els/api/categories/prorefrigeratedcabinets/ProRefrigeratedCabinetsApiController.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+public class ProRefrigeratedCabinetsApiController {
+
+  private final ProRefrigeratedCabinetsService proRefrigeratedCabinetsService;
+  private final InternetLabelService internetLabelService;
+  private final DocumentRendererService documentRendererService;
+
+  @Autowired
+  public ProRefrigeratedCabinetsApiController(
+      ProRefrigeratedCabinetsService proRefrigeratedCabinetsService,
+      InternetLabelService internetLabelService, DocumentRendererService documentRendererService) {
```
Comment (dannybetts) [inline src/main/java/uk/gov/beis/els/api/categories/prorefrigeratedcabinets/ProRefrigeratedCabinetsApiController.java:29]: Nit: I've spotted this strange formatting in quite a few of your controllers if you can put each param on a new line please and review the other ones.

**Review by dannybetts — CHANGES_REQUESTED** [review]
Format controller constructors

File: src/main/java/uk/gov/beis/els/api/categories/prorefrigeratedcabinets/ProRefrigeratedCabinetsApiController.java (hunk 1 above)
Comment (FernandoRamirez-De-Aguilar) [reply src/main/java/uk/gov/beis/els/api/categories/prorefrigeratedcabinets/ProRefrigeratedCabinetsApiController.java]: done

**Review by dannybetts — APPROVED** [review]

**Review by jamesbarnett91 — APPROVED** [review]
LGTM, thanks

**Pull request merged**